### PR TITLE
Deregister hotel mailer

### PR DESCRIPTION
**Pull request by MariethPerez**
Deregister hotel when deleted bookings mailer

### CODE REVIEW

File: app/controllers/admin/hotels_controller.rb [hunk 1]
```diff
@@ -50,4 +51,16 @@ def hotel_params
...
+      @bookings = @hotel.bookings
+        if @bookings.any?
+          @bookings.each do |booking|
+            BookingMailer
+            .with(user: booking.user_id, booking: booking)
+            .booking_deleted
+            .deliver_later
+          end
+        end
+    end
```
Comment (paulotijero) [inline app/controllers/admin/hotels_controller.rb]: Always remember to be careful with tabs

File: app/mailers/booking_mailer.rb [hunk 2]
```diff
@@ -0,0 +1,9 @@
+class BookingMailer < ApplicationMailer
+    default from: 'notifications@myblog.com'
```
Comment (condef5) [inline app/mailers/booking_mailer.rb]: The identation is important :)

File: app/models/booking.rb [hunk 3]
```diff
@@ -2,3 +2,6 @@ class Booking < ApplicationRecord
   belongs_to :room
   belongs_to :user
 end
+
```
Comment (condef5) [inline app/models/booking.rb]: Remove your blanks.

File: app/views/booking_mailer/booking_deleted.html.erb [hunk 4]
```diff
@@ -0,0 +1,17 @@
+<%@room=Room.find(@booking.room_id) %>
```
Comment (condef5) [inline app/views/booking_mailer/booking_deleted.html.erb]: Hey, this line is unnecessary, you can access the properties of the room related in this way:
```ruby
  <dd><%= @booking.room.name %></dd>
```

**Comment by MariethPerez** [comment]
Resolve #49

**Pull request merged**